packages/fxa-react/components/Portal/index.tsx

```diff
@@ -45,9 +45,12 @@ const Portal = ({
       el.setAttribute('role', 'dialog');
     }
 
-    if (id === 'modal') {
+    if (id === 'dialogs' || id === 'modal') {
```

Comment:
These are the only possible IDs right? I think you can just take these out of the `if` if so.

Reply:
I believe there are a total of four: `alert-bar-portal`, `dialogs`, `modal`, and `top-bar`. I'll add `headerId` and `descId` to the other two and then remove the `if`. 👍